src/components/ListItem/ListItem.tsx

```diff
@@ -108,21 +109,22 @@ const ListItem: FC<ListItemProps> & { sizes?: typeof SIZES } = forwardRef(
 
     return (
       <Text
-        element="div"
-        data-testid={dataTestId || getTestId(ComponentDefaultTestId.LIST_ITEM, id)}
+        element="li"
+        data-testid={dataTestId || id}
```

Comment:
So default `data-testid` won't be like this: `list-item_list-item-0`
We already have a solid default id, so I've decided it's better just to reuse it here